atomic_reactor/plugins/post_tag_and_push.py

```diff
@@ -134,18 +137,38 @@ def run(self):
 
                 registry_image = image.copy()
                 registry_image.registry = registry
-                if self.need_skopeo_push():
-                    self.push_with_skopeo(registry_image, insecure, docker_push_secret)
-                else:
-                    self.tasker.tag_and_push_image(self.workflow.builder.image_id,
-                                                   registry_image, insecure=insecure,
-                                                   force=True, dockercfg=docker_push_secret)
-                    defer_removal(self.workflow, registry_image)
+                max_retries = DOCKER_MAX_RETRIES
```

Comment:
This is actually the number of tries, right? For instance, if one decides to set it to 0, no pushes would happen, when one would most likely be expecting 1 push attempt to happen.

Maybe we could create a new constant for this case (instead of change the logic in this patch) since DOCKER_MAX_RETRIES is used for something else in the `core` module and tuning it for one purpose could result in unexpected behavior somewhere else. Another option would be to set `max_tries` to 1+ max_retries or something like that, but then it would be nice to evaluate the consequences of changing DOCKER_MAX_RETRIES value. What do you think?

Reply:
created new constants